lang/pkl/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               java 1.0
+PortGroup               github 1.0
+
+github.setup            apple pkl 0.25.1
+
+java.version            17+
+java.fallback           openjdk17
+
+categories              lang
+maintainers             {sub-pop.net:link @subpop} openmaintainer
+
+license                 Apache-2.0
+
+description             A configuration as code language with rich validation and tooling
+long_description        Pkl (pronounced Pickle) is an embeddable configuration \
+                        language which provides rich support for data templating \
+                        and validation. It can be used from the command line, \
+                        integrated in a build pipeline, or embedded in a program. \
+                        Pkl scales from small to large, simple to complex, ad-hoc \
+                        to repetitive configuration tasks.
+
+checksums               rmd160  1e822e7a57050b392a216d9a808283d42df3801b \
+                        sha256  f1588550d820e226b9c387abcffae50c4c64ee313a872acdefbe2f61b6f60e84 \
+                        size    2769668
+
+if {${os.arch} eq "arm"} {
+patchfiles              graalVm23.patch
+patch.args              -p1
+}
+
+use_configure           no
+
+supported_archs         arm64 x86_64
+
+build.cmd               ./gradlew --info --stacktrace
```

Comment:
I have no knowledge about Java stuff... but the `gradlew` command is provided by the Java installation... or?

Reply:
It's a shell script that ships in the source repository.